src/net/UdpMulticastServer.ts

```diff
@@ -52,7 +55,7 @@ export class UdpMulticastServer {
             const { ips } = this.network.getIpMac(netInterface) ?? { ips: [] };
             await Promise.all(ips.map(async ip => {
                 const iPv4 = isIPv4(ip);
-                await (await this.broadcastChannels.get(netInterface, iPv4)).send(iPv4 ? this.broadcastAddressIpv4 : this.broadcastAddressIpv6, this.broadcastPort, message);
+                await (await this.broadcastChannels.get(netInterface, iPv4)).send(iPv4 ? this.broadcastAddressIpv4 : this.broadcastAddressIpv6, this.broadcastPort, message).catch(err => logger.debug(`${netInterface}: ${err}`));
```

Comment:
logger.warn ?
I think this match the definition of a warning: error but with no expected functional impact

Reply:
Hm ... for Me warn is something where the user should react on but here he can not even do anything. That's why for me maximal info ;-)

Reply:
I adjusted it to info.